esp-hal/src/uart.rs

```diff
@@ -1852,16 +1853,14 @@ mod asynch {
             let mut events_triggered = EnumSet::new();
             for event in self.events {
                 let event_triggered = match event {
-                    RxEvent::RxFifoFull => interrupts_enabled.rxfifo_full().bit_is_clear(),
-                    RxEvent::RxCmdCharDetected => {
-                        interrupts_enabled.at_cmd_char_det().bit_is_clear()
-                    }
-
-                    RxEvent::RxFifoOvf => interrupts_enabled.rxfifo_ovf().bit_is_clear(),
-                    RxEvent::RxFifoTout => interrupts_enabled.rxfifo_tout().bit_is_clear(),
-                    RxEvent::RxGlitchDetected => interrupts_enabled.glitch_det().bit_is_clear(),
-                    RxEvent::RxFrameError => interrupts_enabled.frm_err().bit_is_clear(),
-                    RxEvent::RxParityError => interrupts_enabled.parity_err().bit_is_clear(),
+                    RxEvent::FifoFull => interrupts_enabled.rxfifo_full().bit_is_clear(),
+                    RxEvent::CmdCharDetected => interrupts_enabled.at_cmd_char_det().bit_is_clear(),
+
```

Comment:
```suggestion
```

Reply:
What's the suggested change? I just see an empty line? :D

Reply:
Haha, that's the suggested change - remove the empty line :D

Reply:
rustfmt put it there, so I'll leave it be :D